### ISSUE TITLE
Setup commit linter and checker

### ISSUE THREAD

**Issue by UmangBR** [amosproj/amos2022ss03-turtlebot-fleet-management#49]
## User story
1. As a SD
2. I want to have a commit linter and checker
3. So that I can ensure that the code on main is formatted properly and the commit msgs contain at least some basic information

## Progress updates
* What i have done so far
* Step 1
* Step 2
* ...

## Acceptance criteria
* Code on main is formatted properly
* Commit msgs contain at least some basic information
* ...

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Sebastian2023** [comment]
https://github.com/github/super-linter
+ 
https://github.com/GsActions/commit-message-checker
